content/blog/18-19/chat2.md

```diff
@@ -2,50 +2,75 @@
 title: "State of the Chat"
 created_at: 14-10-2018
 description: De langverwachte beslissing
-author: Wout Schellaert
+author: het Zeusbestuur
 ---
 
-Hey Zeusjes
+Laat ons de **tl;dr** eerst geven: vanaf nu is [Mattermost](mattermost.zeus.gent) het officiële chat-kanaal van Zeus WPI. Het Slackteam blijft echter wel bestaan voor zij die willen.
 
-Laat ons de **tl;dr** eerst geven: het is Mattermost geworden.
+# Waarom veranderen?
 
-Nu, wij kunnen (en willen) niemand forceren om ons in die keuze te volgen. Het bestuur zal voor alle chatting migreren naar Mattermost, en hopen dat jullie dat samen met ons ook doen. Nieuwe leden zullen we ook naar daar verwijzen.
+In het begin van de vakantie hebben we al een [blogpost](/blog/18-19/chat/) gemaakt waarin we zeggen waarom we willen afstappen van ons huidig chatplatform. De grote tekortkomingen waren, kort samengevat:
+
+- Een gebrek aan backlog: je kunt enkel de laatste 10.000 berichten bekijken
+- Een gebrek aan 'hackbaarheid': Slack is closed-source en niet self-hosted, dus wij hangen voor 100% af van het bedrijf achter Slack. Op de gratis versie is het aantal integraties ook beperkt, waardoor we zelf geen nieuwe coole features konden toevoegen.
+
+# Waarom Mattermost?
+
+
+Er zijn veel alternatieven uitgeprobeerd, maar van alle applicaties was Mattermost die waar we het meest tevreden van waren.
```

Comment:
is it really?

Reply:
Ik al zeker wel

Reply:
Ja? Als er een applicatie was geweest waar we tevredener van waren dan ging het die worden?

Reply:
@rien ik had het op 'veel alternatieven'

Reply:
Ah, ja ook. Zullip en Rocket.chat hebben we gedeployed op de server. Matrix hebben we geprobeerd, maar bleek relatief complex te zijn (+ er waren enkele minpunten aan de interface waardoor het de moeite niet echt waard was).